config/common/files/.github/PULL_REQUEST_TEMPLATE.md.mustache

```diff
@@ -1 +0,0 @@
-We are not accepting new Pull Requests for the time being. Please raise an issue or contact us in the [support community]({{supportInfo}}) instead.
```

Comment:
Any reason we remove these?

Reply:
To encourage people to send PRs, we'll do the heavy work of porting it over to the generator. Asking them to submit PRs here is not always easy for them